pyannote/audio/applications/change_detection.py

```diff
@@ -318,11 +335,19 @@ def main():
 
         purity = float(arguments['--purity'])
 
+        metric_type = arguments['--metric_type']
+        if not metric_type:
+            metric_type = SEGMENTATION_METRIC_TYPE
+
         application = SpeakerChangeDetection.from_train_dir(
             train_dir, db_yml=db_yml, training=False)
+
         application.device = device
         application.batch_size = batch_size
         application.purity = purity
+        application.metric_type = metric_type
+        application.pool_ = mp.Pool(mp.cpu_count())
```

Comment:
Why is this needed?  `pool_` is already instantiated in (inherited) `SpeechActivityDetection.validate_init`, isn't it?

Reply:
Actually, this is a bug. Checking `SpeechActivityDetection.validate_init`, it is returning before setting `pool_` ... Should I move that line up? Why is it returning when `self.feature_extraction_` is a `Precomputed` instance?

Reply:
Oh. Right. This is indeed a bug. I just pushed a fix.